tests/docker/build.sh

```diff
@@ -44,7 +44,7 @@ ninja install
 
 # libgav1
 cd
-git clone -b v0.18.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
+git clone -b v0.19.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
```

Comment:
Please also add an item to CHANGELOG.md:
https://github.com/AOMediaCodec/libavif/blob/main/CHANGELOG.md#changed

Reply:
Done